src/app/pages/Notifications/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const formatGeneralDate = (date: number | string) =>
```

Comment:
Please try using `formatDate` with appropriate format string (see https://date-fns.org/v4.1.0/docs/format for all options)